pkgs/top-level/all-packages.nix

```diff
@@ -36965,10 +36965,6 @@ with pkgs;
 
   zita-njbridge = callPackage ../applications/audio/zita-njbridge { };
 
-  zola = callPackage ../applications/misc/zola {
-    inherit (darwin.apple_sdk.frameworks) CoreServices;
-  };
-
```

Comment:
Doesn't this prevent zola from being installable via standard methods?

Did you mean to only remove the inherit line?

Reply:
It doesn't, see [this](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories):

> Packages found in the named-based structure do not need to be explicitly added to the `top-level/all-packages.nix` file unless they require overriding the default value of an implicit attribute.

Reply:
Ah interesting! Thanks.